teacher.rb

```diff
@@ -1,12 +1,14 @@
-require './person'
+require_relative 'person'
 
 class Teacher < Person
-    def initialize (specialization, name, age, id, parent_permission="unknown")
-        @specialization = specialization
-        super(name, age, id, parent_permission)
-    end
+  attr_reader :specialization
 
-    def can_use_services?
-        return true
-    end
+  def initialize(specialization, age, name, parent_permmission: true)
+    super(age, name, parent_permmission: parent_permmission)
+    @specialization = specialization
+  end
```

Comment:
- [ ] Good job making the requested changes, however a user is still unable to create a `Teacher` in the app. This is because in the `initialize` method, there is a typo with `parent_permisson` it is spelled `parent_permmission`. Kindly fix it by renaming all instances of `parent_permmission` to `parent_permission` to fix this.

e.g

```
def initialize(specialization, age, name, parent_permission: true)
    super(age, name, parent_permission: parent_permission)
    @specialization = specialization
  end
```

Screenshot of error
![image](https://github.com/Reggeditt/school-library/assets/81932478/1b793ce1-ba11-442a-8ce7-ddb4e334bdc6)

### Note
- [ ] Once you fix this you should be able to create a rental, kindly double check to make sure before resubmitting for review